toolchain/build-toolchain.sh

```diff
@@ -2,24 +2,39 @@
 # $1 - toolchain target (e.g. arm-phoenix)
 # $2 - toolchain install absolute path (i.e. no "." or ".." in the path)
 
-
 set -e
 
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
 log() {
     echo -e "\e[35;1m$*\e[0m"
 }
 
-if [ -z "$1" ]; then
-    echo "No toolchain target provided! Abort."
-    echo "officially supported targets: arm-phoenix i386-pc-phoenix riscv64-phoenix"
+# targets for libphoenix and phoenix-rtos-kernel for installing headers
+declare -A TOOLCHAN_TO_PHOENIX_TARGET=(
+    [arm-phoenix]="armv7a7-imx6ull"
```

Comment:
I'm not sure what of kernel headers and libc sources is exactly used by gcc while building. I'm a little worry that libstdc++-v3 use some defines/libs and it will be invalid for other arm target.

Just a side worry, I suppose it's safe for now, and provide good more valid compiler than previous approach.

Reply:
I've answered that in separate response below.

For full-featured binary toolchain release we should be able to compile libc for all multilib variants (obviously not possible right now  :)).